packages/myst-frontmatter/src/frontmatter/frontmatter.spec.ts

```diff
@@ -47,25 +47,28 @@ const TEST_BIBLIO: Biblio = {
 };
 
 const TEST_THEBE: Thebe = {
-  useBinder: false,
-  useJupyterLite: false,
-  requestKernel: true,
-  binderOptions: {
-    binderUrl: 'https://my.binder.org/blah',
+  lite: false,
+  binder: {
+    url: 'https://my.binder.org/blah',
     ref: 'HEAD',
     repo: 'my-repo',
-    repoProvider: 'github',
+    provider: 'github',
   },
-  serverSettings: {
-    baseUrl: 'http://localhost:8888',
-    token: 'test-secret',
-    wsUrl: 'ws://localhost:8888',
-    appendToken: true,
+  server: {
```

Comment:
I modified this from `serverSettings` to just `server`. This is equivalent to `binder` and `local` as opposed to `binderSettings` and `localSettings`.

Reply:
I was toying with that one when writing this but yeah there is no good reason to keep the `serverSettings` here - it's a jupyterlab naming convention for that data fragment but no need to keep it here

Reply:
so maybe we can go with `url` for `baseUrl` after all too

Reply:
Ok, I'll update to these simpler names.

If we also want to cater to folks intimately familiar with jupyterlab, we could leave these other names as aliases as well 🤷‍♀️ but I guess there's also the `path` vs `sessionName` stuff which may not just be an alias but actually have a slightly different meaning.